docs/contributing/multimodal/adding_new_models.md

```diff
@@ -0,0 +1,71 @@
+# Multimodal Models in vLLM Spyre
+
+In order to understand how to get multimodal models running through vLLM Spyre, it is important to understand the differences between how models are implemented in vLLM & vLLM Spyre. To illustrate this, we use `llava_next` as an example, because `granite vision` is the only multimodal model currently supported.
```

Comment:
it isn't immediately obvious to the uninitiated that granite-vision is a llava-next model

Reply:
Yeah, I will add a point clarifying this. In case someone finds this comment later - architecturally, the only difference between normal llava next and granite vision is that:

- llava next - you throw your images through the visual encoder, take the result of one layer, and use that as image features

- granite vision - same thing, but you take the result of multiple layers and concatenate them to form the image features

since it's such a small change, it's implemented in everything (transformers, vllm, llama cpp, etc) as an instance of llava next